internal/names.go

```diff
@@ -24,9 +24,13 @@ const (
 
 	// Namespace is the reverse DNS namespace used with labels and CNAB custom actions.
 	Namespace = "com.docker.app."
+	// CnabNamespace is the namespace used with the CNAB well known custom actions
+	CnabNamespace = "io.cnab."
 
-	// ActionStatusName is the name of the custom "status" action
-	ActionStatusName = Namespace + "status"
+	// ActionDeprecatedStatusName is the name of the deprecated custom "status" action
+	ActionDeprecatedStatusName = Namespace + "status"
```

Comment:
perhaps `ActionStatusNameDeprecated` (or `..Legacy`) to make it group with the new one

Reply:
Indeed. I renamed the constant